src/main/scala/sigmastate/serialization/OpCodes.scala

```diff
@@ -109,7 +109,7 @@ object OpCodes extends ValueCodes {
   val SliceCode            : OpCode = (LastConstantCode + 68).toByte
   val FilterCode           : OpCode = (LastConstantCode + 69).toByte
   val TreeLookupCode       : OpCode = (LastConstantCode + 70).toByte
-  val StringConcatCode     : OpCode = (LastConstantCode + 71).toByte
+  //  FREE                 : OpCode = (LastConstantCode + 71).toByte
```

Comment:
@greenhat why not to change TreeModificationsCode?

Reply:
@ergomorphic I've prematurely switched into "don't break compatibility" mode. Which does not make any sense at least until probably Testnet2. :) Fixed.